packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -306,12 +297,13 @@ const Dropdown = ({
           error={error}
           selected={selectedItem !== null}
         >
-          <Input
+          <Field
             readOnly
-            placeholder={label}
             disabled={disabled}
             selected={selectedItem !== null}
             {...getInputProps()}
```

Comment:
I guess we should move this getInputProps to the last position of the Field props. This way we could overwrite isOpen or label if we need. Does it make sense?

``` js
<Field
  readOnly
  disabled={disabled}
  selected={selectedItem !== null}
  isOpen={isOpen}
  label={label}
  {...getInputProps()}
/>
```